history-of-quakers/modernized/28-appendix.adoc

```diff
@@ -7,9 +7,8 @@ and the plainness of its expressions.
 It is the more like the man,
 and carries the greater evidence of the honesty and integrity of the relater:
 
-The account of the sufferings of Richard Seller,
-of Keinsey a fisherman who was pressed in Scarborough-Piers,
-in the time of the two last engagements between the Dutch and English, in the year 1665.
+[.blurb]
+=== The account of the sufferings of Richard Seller, of Keinsey a fisherman who was pressed in Scarborough-Piers, in the time of the two last engagements between the Dutch and English, in the year 1665.
```

Comment:
Also, I wasn't sure if this should be embedded, I'm thinking now that it probably should be.

Reply:
Yeah, let's probably embed this.

Reply:
I labeled it as a testimony....not sure if that's right!